rlbot.cfg

```diff
@@ -129,3 +129,6 @@ participant_loadout_config_8 = None
 participant_loadout_config_9 = None
 
 
+[Scripts]
+script_config_0=src/test/python/agents/script/sample_script.cfg
+script_config_1=src/test/python/agents/script/sample_script.cfg
```

Comment:
How come the sample script is listed twice in the default cfg?

Reply:
I wanted to prove to myself that it would successfully load two scripts, but was too lazy to actually make two unique test scripts.